proto/pdpb.proto

```diff
@@ -558,3 +562,25 @@ message GetOperatorResponse {
     OperatorStatus status = 4;
     bytes kind = 5;
 }
+
+message SetGCSafePointLimitRequest {
+    RequestHeader header = 1;
+    // UUID v4
+    bytes uuid = 2;
+    uint64 safe_point = 3;
+    uint64 ttl = 4;
+}
+
+message SetGCSafePointLimitResponse {
+    ResponseHeader header = 1;
+}
+
+message GetGCSafePointLimitRequest {
+    RequestHeader header = 1;
+}
+
+message GetGCSafePointLimitResponse {
+    RequestHeader header = 1;
+    uint64 safe_point = 2;
+    uint64 min_safe_point_limit = 3;
```

Comment:
Why not extend the `GetGCSafePointResponse` instead?

Reply:
Oh, sorry. I misunderstood the document here.